containers/clojure/README.md

```diff
@@ -0,0 +1,134 @@
+# Clojure
+
+## Summary
+
+*Develop Clojure applications. Includes the Calva and clj-kondo extensions.*
+
+| Metadata                    | Value                 |
+|-----------------------------|-----------------------|
```

Comment:
Please add the "Contributors" metadata point along with the people or GitHub teams we should "@" into any questions or issues with the definition.

Reply:
I missed this item entirely, thank you! I've added both myself and @matthewferry to the list of contributors.